docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  db:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: pw
```

Comment:
related to the previous comment, having secrets like password checked in is generally not a good practice. You could use the -e flag for docker compose to specify a input variable, then here you can use the input variable like so:

```
environment:
    POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
```

Reply:
It's a local testing db, I don't think it's a problem to check in the password for this?
We have the local testing db to mimic remote prod db. That way we can test our migrations in the local testing db before pushing to prod.

Reply:
Relating to the previous comment about `.env.local`. In this case it's now more about having a single source of truth, since the password is already defined in `.env.local` we should use values defined there and not have to maintain the same variable in multiple places.